source/manifest.json

```diff
@@ -1,4 +1,10 @@
 {
+  "applications": {
+    "gecko": {
+      "id": "theoldreader@knyar",
+      "strict_min_version": "53.0"
+    }
+  },
```

Comment:
If I remember correctly, this will fail to validate in Chrome; also, might not be needed in FF (for publishing).
But my knowledge of FF is shaky here.
Split manifest is one of the reasons I am wary of a 100% unified codebase.